scripts/improve-localization.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const path = require('path');
+const readline = require('readline');
+
+const PATH_TO_LANG = path.resolve(__dirname, '..', 'app', 'lang');
+
+const getLanguage = () => {
```

Comment:
We really should have done this for the old find-strings.js script, but could you add a block comment to the top of this script file explaining how to call it and what it does? It looks nice overall!

Reply:
Thanks! I've added the indications in the **LOCALIZATION.md** file to make it more natural.